src/core/adapters/plane.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Plane adapter
+ *
+ * The adapter extracts ticket information from the markup.
+ */
+
+import { match } from "micro-match";
+
+import type { TicketData } from "../types";
+import { $text } from "./dom-helpers";
+
+async function scan(url: URL, document: Document): Promise<TicketData[]> {
+  const checkForPlane = document.querySelector("[property~='og:title'][content]");
+  if (!checkForPlane || !checkForPlane.content.startsWith("Plane")) return [];
+
+  let { issueType } = match("/:workspace/projects/:projectId/:issueType/:id/", url.pathname);
+  if (issueType == "issues") {
+    issueType = "feature";
+  }
+  const pageTitle = $text("title", document);
+  const extract = pageTitle && pageTitle.match(/([A-Z]+-\d+)\s(.+)/);
+  const id = extract && extract[1];
+  const title = extract && extract[2];
+
+  if (!id) return [];
```

Comment:
I think we can skip some checks and the extra return if we use the filtering in the end. So this should be enough:

```suggestion
  const pageTitle = $text("title", document);
  const extract = pageTitle?.match(/([A-Z]+-\d+)\s(.+)/);
  const id = extract && extract[1]
  const title = extract && extract [2];
```

But more importantly: Why is it necessary to extract the `ID` again, if it would already have been part of the URL?

Reply:
The url includes a uuid and not the "issue id", I've changed the naming of the match variables.